src/Data/Reason.hs

```diff
@@ -52,6 +52,9 @@ instance (Data p, Data (Ann p), Data (Var p), Pretty (Var p)) => Reasonable Cons
 instance (Spanned (Ann p), Data p, Data (Ann p), Data (Var p), Pretty (Var p)) => Reasonable Toplevel p where
   blame _ = string "the" <+> highlight "declaration"
 
+instance (Spanned (Ann p), Pretty (Var p)) => Reasonable Binding p where
```

Comment:
Not a required change, but I almost wonder if we should move some of these instances to `Syntax`.